### PR TITLE
Feat/persist sort order for flex table

### PR DESCRIPTION
**Pull request by gustav-pettersson-nordnet**

### CODE REVIEW

File: src/Molecules/FlexTable/shared/FlexTableProvider/FlexTableProvider.types.ts [hunk 1]
```diff
@@ -26,6 +31,11 @@ export type FlexTableState = {
    * @default false
    */
   expandable: boolean;
+  /**
+   * Remember last sorted column. The last sorted column will be stored in local storage and be the initial sorted column.
+   * @default false
+   */
+  persistSortingOrder: boolean;
```
Comment (gustav-pettersson-nordnet) [inline src/Molecules/FlexTable/shared/FlexTableProvider/FlexTableProvider.types.ts:38]: persistSortingOrder is part of FlexTableContextProps what I can see but I still get typescript error about It.

File: src/Molecules/FlexTable/shared/persistedSortOrder.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { SortOrder } from '../Header/HeaderContent/HeaderContent.types';
+import { persistedSortOrderLocalStorageKey } from './constants';
+
+export const tableHasSavedPersistedSortOrder = (tableId: string | undefined) => {
+  if (!tableId) {
+    return false;
+  }
+
+  const sortedSortOrders = localStorage.getItem(persistedSortOrderLocalStorageKey);
```
Comment (jesperwiner) [inline src/Molecules/FlexTable/shared/persistedSortOrder.ts:9]: add a try/catch around all access to local storage please

File: src/Molecules/FlexTable/shared/persistedSortOrder.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  if (!parsed) {
+    return false;
+  }
+  return !!parsed[tableId];
+};
+
+export const getPersistedSortOrder = (
+  tableId: string | undefined,
+): { columnId: string; sortOrder: SortOrder } | null => {
+  if (tableId && tableHasSavedPersistedSortOrder(tableId)) {
```
Comment (jesperwiner) [inline src/Molecules/FlexTable/shared/persistedSortOrder.ts:24]: here too, we have found that some browsers will throw exceptions there that bubble all the way to the oops page

**Review by jesperwiner — CHANGES_REQUESTED** [review]

**Review by almightybruce — APPROVED** [review]
Looks good. I think you should be consistent between `sortOrder` and `sortingOrder`. And maybe `tableHasSavedPersistedSortOrder` can lose the _saved_

**Review by jesperwiner — APPROVED** [review]

**Pull request merged**